charts/grafana/templates/image-renderer-deployment.yaml

```diff
@@ -109,6 +109,24 @@ spec:
           volumeMounts:
             - mountPath: /tmp
               name: image-renderer-tmpfs
+            {{- range .Values.imageRenderer.extraConfigmapMounts }}
+            - name: {{ tpl .name $root }}
+              mountPath: {{ tpl .mountPath $root }}
```

Comment:
Could you please explain, why tpl is required here?

Reply:
In this case it's just to remain consistent with the same options in the Grafana deployment. I'm not sure why templating was only enabled for extraConfigmapMounts and not extraSecretMounts/extraVolumeMounts, but it was introduced in this PR - https://github.com/grafana/helm-charts/pull/1333.

Reply:
Thanks for clarification!